### DISCUSSION TITLE
[Bug]: Opening Diablo Immortal Apps made other apps' sound very low

### DISCUSSION THREAD

**Discussion by danielggani** [PlayCover/PlayCover#435] [General]
### Describe the bug

PlayCover version: PlayCover 1.1.1
macOS version: 12.6 Monterey

App Name: Diablo Immortal
App Version: Diablo Immortal 1.5.4
What you have tried so far: Restart my macbook, Re-install

Description: So everytime I Opened Diablo Apps from playcover, other apps' sound like (e.g. spotify/youtube, etc) became low that I barely hear nothing. **If i can give you an analogy, the case is the same as if u picking up a call (whatsapp) from desktop/laptop suddenly other apps sound became low, but this happened when i open diablo immortal**

### Steps to reproduce

1. Launch Playcover
2. Launch Diablo Immortal
3. Try to compare before and after launching diablo immortal (use background music or something from spotify/youtube)

### Expected behaviour

Other apps wont be affected when diablo immortal launched and still have normal sound

### What version of PlayCover are you using?

1.1.1

### What version of macOS are you using?

Monterey (macOS 12)

### Issue Language

- [X] Yes my issue is written in English

**Comment by IsaacMarovitz** [comment]
Are you using headphones or other external/Bluetooth devices? If so, that could cause your sampling rate to drop. You can also try altering your settings in Audio MIDI Setup.

**Comment by danielggani** [comment]
Hi Isaac, im currently using mouse and keyboard devices, however ive tried to turn the bluetooth off and seems nothing happened. Tried to alter settings in audio midi setup also didn't work. 

From my observation, it must be related with in-game mic voice. (I've tried to enabled mic on and suddenly other apps' sound **became normal and loud**, however if i disabled and mic off the sound from other apps went gone again)

Do you have any other solutions ? thanks

**Comment by danielggani** [comment]
Hi @IsaacMarovitz following up last query. Do we have any update on this ?

**Comment by danielggani** [comment]
@IsaacMarovitz @lixin9311 @hayatofujii @amirsaam Following up again for the last query, do we have any update on this ?

**Comment by danielggani** [comment]
> @IsaacMarovitz @lixin9311 @hayatofujii @amirsaam Following up again for the last query, do we have any update on this ?

FU again

**Comment by amirsaam** [comment]
Hey buddy, sorry but I am not an expert and I dunno what can fix your issue.
  
I can give you some suggestion
- Install ACE Audio Driver by installing [SoundSource](https://rogueamoeba.com/soundsource/) (You only need the free version)
- Trying PlayCover [Nightly](https://github.com/PlayCover/PlayCover/actions/runs/3176979258)
- If only above suggestion didn't work, open your mic in the game while disabling it from macOS System Preferences
  
I am afraid that this issue is not from PlayCover but on how Diablo itself handles audio input and output.

**Comment by valoryn** [comment]
In my case SoundSource not worked in first test but when I restart app and repair ACE driver on app while game is running, It fixed the issue.

**Comment by amirsaam** [comment]
Uh yeah, SoundSource needs a restart.
Glad it worked!
